### PR TITLE
MIME4J-233 Fix ByteBufferInputStream read implementation

### PR DESCRIPTION
**Pull request by randomnetcat**

### CODE REVIEW

**Review by mbaechler — APPROVED** [review]
It looks good, thank you for this work

File: mbox/src/test/java/org/apache/james/mime4j/mboxiterator/ByteBufferInputStreamTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package org.apache.james.mime4j.mboxiterator;
```
Comment (chibenwa) [inline mbox/src/test/java/org/apache/james/mime4j/mboxiterator/ByteBufferInputStreamTest.java:1]: We are missing the compulsory Apache license header. Can you add it like in the other files?

**Review by chibenwa — APPROVED** [review]

File: mbox/src/test/java/org/apache/james/mime4j/mboxiterator/ByteBufferInputStreamTest.java (hunk 1 above)
Comment (randomnetcat) [reply mbox/src/test/java/org/apache/james/mime4j/mboxiterator/ByteBufferInputStreamTest.java:19]: Done.

**Review by chibenwa — APPROVED** [review]
Thanks!

**Comment by chibenwa** [comment]
Why did you close this?

**Comment by randomnetcat** [comment]
Because it appears to be in master already unless I'm very confused https://github.com/apache/james-mime4j/commit/4861043e44d34d8cfcf16990720031083792b178

**Comment by chibenwa** [comment]
Ok fair.

Somebody merged this work on master, and I did not get any notice about it.

Sorry for the confusion, everything is fine. Thanks for the contribution.

Cheers,

Benoit

**Pull request closed without merging**